### PR TITLE
Add solr

### PR DESCRIPTION
**Pull request by howareyouman**

### CODE REVIEW

**Comment by towelenee** [comment]
Looks good to me

File: src/main/java/com/w6/controller/EndpointController.java [hunk 1]
```diff
@@ -14,18 +18,39 @@
...
+            solrClient.uploadDataToSolr(text);
+        } catch (SolrServerException ex) {
+            Logger.getLogger(EndpointController.class.getName()).log(Level.SEVERE, null, ex);
+        }
+        return new ModelAndView(UPLOAD_VIEW);
+    }
 
-    @RequestMapping(value = "parse", method = RequestMethod.POST)
-    public ModelAndView parse(@RequestParam("text") String text) throws IOException
+    @RequestMapping(value = "parse", method = RequestMethod.GET)
```
Comment (towelenee) [inline src/main/java/com/w6/controller/EndpointController.java:38]: Why input and upload is a different pages?

Comment (howareyouman) [inline src/main/java/com/w6/controller/EndpointController.java:38]: because upload is page for success loading

**Comment by towelenee** [comment]
Looks good to me please merge all pull requests

**Pull request merged**